.github/workflows/progress-tracker.yml

```
@@ -0,0 +1,121 @@
+name: Progress Tracker
+
+env:
+  # Update these environment variables every year
+  FILTER_LABEL: '2020 chapter'
+  TRACKER_ISSUE_NUMBER: 989
+  TOTAL_TRACKED_TASKS: 10
+
+on:
+  workflow_dispatch:
+  issues:
+    types:
+      - edited
+
+jobs:
+  track-progress:
+    name: Track Progress
+    if: github.repository == 'HTTPArchive/almanac.httparchive.org'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v2
+        if: github.event_name == 'workflow_dispatch' || contains(github.event.issue.labels.*.name, env.FILTER_LABEL)
+        with:
+          script: |
+            const filterLabel = process.env.FILTER_LABEL
+            const trackerIssueNumber = process.env.TRACKER_ISSUE_NUMBER
+            const totalTrackedTasks = parseInt(process.env.TOTAL_TRACKED_TASKS, 10)
+
+            const parseTasks = text => {
+              const tasks = []
+              text.split('\n').forEach(line => {
+                if (/^(\s{2,})?([*+-]|\d+\.)\s+\[ \]\s+\S+/.test(line)) {
+                  tasks.push('')
+                }
+                if (/^(\s{2,})?([*+-]|\d+\.)\s+\[[xX]\]\s+\S+/.test(line)) {
+                  tasks.push('☑')
+                }
+              })
+              return tasks
+            }
+
+            const parseLinks = text => {
+              const links = {}
+              text.split('\n').forEach(line => {
+                const linkMap = {
+                  draft: /^\[~draft-doc\]:\s*(?<link>\S+)/,
+                  sql: /^\[~sql-dir\]:\s*(?<link>\S+)/,
+                  results: /^\[~results-sheet\]:\s*(?<link>\S+)/
+                }
+                for (const [k, v] of Object.entries(linkMap)) {
+                  const m = line.match(v)
+                  if (m) {
+                    links[k] = m.groups.link
+                  }
+                }
+              })
+              return links
+            }
+
+            const parseTeam = text => {
+              const team = {}
+              text.split('\n').forEach(line => {
+                if (/\|\s*\[Sheet\]\[~results-sheet\]\s*\|/.test(line)) {
+                  const [, authors, reviewers, analysts] = line.split('|')
+                  team.authors = [...authors.matchAll(/@(?<username>\w+)/g)].map(u => u.groups.username)
+                  team.reviewers = [...reviewers.matchAll(/@(?<username>\w+)/g)].map(u => u.groups.username)
+                  team.analysts = [...analysts.matchAll(/@(?<username>\w+)/g)].map(u => u.groups.username)
+                }
+              })
+              return team
+            }
+
+            const formatLinks = links => {
+              const icons = {
+                draft: '📄',
+                sql: '🔍',
+                results: '📊'
+              }
+              return Object.entries(links).filter(l => l[1] && l[1] != '#').map(l => `[${icons[l[0]] || l[0]}](${l[1]})`).join(' ')
```

```suggestion
              return Object.entries(links).filter(l => l[1] && l[1] != '#' && l[1] != 'https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020/').map(l => `[${icons[l[0]] || l[0]}](${l[1]})`).join(' ')
```

Requested by @rviscomi as SQL is linking to SQL dir and not using `#`

---

I think I have taken care of it, thanks for pointing this out.